go.mod

```diff
@@ -0,0 +1,19 @@
+module github.com/rancher/gke-operator
+
+go 1.13
```

Comment:
Is version `1.13` a requirement, or can we use a newer one?

Reply:
the EKS one was written in 1.13. to be honest I have not had the time to investigate if there is a solid reason for 1.13 vs a later one.

Reply:
Not a problem.
@rmweir any objection to `1.15`?

Reply:
At the time I wrote it, I believe rancher was at 1.13: https://github.com/rancher/rancher/commit/3d2e1ec9f6ff204ce984882145a179ec5155d230. You could probably use 1.14 and we can make a ticket to upgrade eks-operator.